analysis/analysis-api/src/org/jetbrains/kotlin/analysis/api/components/KtCallResolver.kt

```diff
@@ -52,5 +53,6 @@ public interface KtCallResolverMixIn : KtAnalysisSessionMixIn {
 }
 
 private inline fun <reified PSI : KtElement> unresolvedKtCallError(element: PSI): Nothing {
-    error("${PSI::class.simpleName} should always resolve to a KtCallInfo\nelement: ${element::class.simpleName}\ntext:\n${element.getElementTextInContext()}")
+    throw KotlinExceptionWithAttachments("${PSI::class.simpleName} should always resolve to a KtCallInfo")
```

Comment:
The `element::class.simpleName` is useful information too (and it's not a secret 😃). Let's move it to the error message string. For the attachment name, we should use something static, e.g., `elementText`